### ISSUE TITLE
walkthrough/k8s201.html error in rendered text

### ISSUE THREAD

**Issue by davidwalter0** [kubernetes/kubernetes#11696]
The source has the following:

```console
$ export SERVICE_IP=$(kubectl get service nginx-service -o=template -t={{.spec.clusterIP}})
$ export SERVICE_PORT=$(kubectl get service nginx-service -o=template '-t={{(index .spec.ports 0).port}}')
$ curl http://${SERVICE_IP}:${SERVICE_PORT}
```
but on the support kubernetes.io site at :

http://kubernetes.io/v1.0/docs/user-guide/walkthrough/k8s201.html

The template has been omitted.
```
$ export SERVICE_IP=$(kubectl get service nginx-service -o=template -t=)
$ export SERVICE_PORT=$(kubectl get service nginx-service -o=template '-t=')
$ curl http://${SERVICE_IP}:${SERVICE_PORT}
```

**Comment by vishh** [comment]
cc @lavalamp

**Comment by lavalamp** [comment]
@krousey @thockin Looks like a problem with the markdown renderer

**Comment by krousey** [comment]
My guess is that by using redcarpet to render and replacing fenced code blocks with `{% highlight <lang> %}` directives, the Go template directives need to be escaped as to not be gobbled by redcarpet.

**Comment by thockin** [comment]
looking at it.

**Comment by thockin** [comment]
Uggh, ok I think adding {% raw %} to all code blocks will do the trick.
Will test a bit.

**Comment by PeterLamar** [comment]
Looks like solution is merged with #11720. Close issue?
